static/.htaccess

```diff
@@ -308,50 +308,25 @@ RewriteRule ^docs/0.25/components/zeebe/tasklist-user-guide /docs/0.25/component
-RewriteRule ^docs/components/optimize/userguide/process-analysis/report-analysis/configure-reports(\/?)$ /optimize/userguide/process-analysis/report-analysis/configure-reports/ [R=301,L]
-RewriteRule ^docs/components/optimize/userguide/process-analysis/report-analysis/view-mode(\/?)$ /optimize/userguide/process-analysis/report-analysis/view-mode/ [R=301,L]
-RewriteRule ^docs/components/optimize/userguide/process-analysis/filters(\/?)$ /optimize/userguide/process-analysis/filters/ [R=301,L]
-RewriteRule ^docs/components/optimize/userguide/process-analysis/metadata-filters(\/?)$ /optimize/userguide/process-analysis/metadata-filters/ [R=301,L]
-RewriteRule ^docs/components/optimize/userguide/process-analysis/instance-state-filters(\/?)$ /optimize/userguide/process-analysis/instance-state-filters/ [R=301,L]
-RewriteRule ^docs/components/optimize/userguide/process-analysis/flow-node-filters(\/?)$ /optimize/userguide/process-analysis/flow-node-filters/ [R=301,L]
-RewriteRule ^docs/components/optimize/userguide/process-analysis/process-instance-filters(\/?)$ /optimize/userguide/process-analysis/process-instance-filters/ [R=301,L]
-RewriteRule ^docs/components/optimize/userguide/process-analysis/variable-filters(\/?)$ /optimize/userguide/process-analysis/variable-filters/ [R=301,L]
-RewriteRule ^docs/components/optimize/userguide/decision-analysis/decision-analysis-overview(\/?)$ /optimize/userguide/decision-analysis/decision-analysis-overview/ [R=301,L]
-RewriteRule ^docs/components/optimize/userguide/decision-analysis/decision-report(\/?)$ /optimize/userguide/decision-analysis/decision-report/ [R=301,L]
-RewriteRule ^docs/components/optimize/userguide/decision-analysis/decision-filter(\/?)$ /optimize/userguide/decision-analysis/decision-filter/ [R=301,L]
-RewriteRule ^docs/components/optimize/userguide/additional-features/alerts(\/?)$ /optimize/userguide/additional-features/alerts/ [R=301,L]
-RewriteRule ^docs/components/optimize/userguide/additional-features/event-based-processes(\/?)$ /optimize/userguide/additional-features/event-based-processes/ [R=301,L]
-RewriteRule ^docs/components/optimize/userguide/additional-features/export-import(\/?)$ /optimize/userguide/additional-features/export-import/ [R=301,L]
-RewriteRule ^docs/components/optimize/userguide/additional-features/footer(\/?)$ /optimize/userguide/additional-features/footer/ [R=301,L]
-RewriteRule ^docs/components/optimize/userguide/additional-features/variable-labeling(\/?)$ /optimize/userguide/additional-features/variable-labeling/ [R=301,L]
+
+## these rules cover all pages that were moved from /docs/components/optimize into /optimize/components:
+RewriteRule ^docs/components/optimize/what-is-optimize(\/?)$ /optimize/components/what-is-optimize/ [R=301,L]
+RewriteRule ^docs/components/optimize/userguide/(.*)$ /optimize/components/userguide/$1 [R=301,L]
```

Comment:
I was not taking advantage of the fact that .htaccess rules are regular expressions....this rule captures all children of docs/components/optimize/userguide so that we don't have to declare them all independently.